azure-pipelines.yml

```diff
@@ -45,7 +45,7 @@ jobs:
   - script: |
       brew install flex bison cmake python@3
       python3 -m pip install -U pip setuptools
-      python3 -m pip install --user 'Jinja2>=2.9.3' 'PyYAML>=3.13' pytest 'sympy>=1.3'
+      python3 -m pip install --user 'Jinja2>=2.9.3' 'PyYAML>=3.13' pytest 'sympy>=1.3,<1.6'
```

Comment:
the changes from `travis.yaml` should be applied here too:
```
  make -j 2;
  if [ $? -ne 0 ]; then
      make VERBOSE=1;
  fi
  env CTEST_OUTPUT_ON_FAILURE=1 make test
```
imho they are useful